src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -105,21 +163,28 @@ function prolong2boundaries!(cache, u,
                              mesh::P4estMesh{2},
                              equations, surface_integral, dg::DG)
   @unpack boundaries = cache
-
-  size_ = (nnodes(dg), nnodes(dg))
+  idx_one = 1
+  idx_end = nnodes(dg)
 
   @threaded for boundary in eachboundary(dg, cache)
+    # Copy solution data from the element on a case-by-case basis to get
+    # the correct face and orientation.
     element       = boundaries.element_ids[boundary]
     node_indices  = boundaries.node_indices[boundary]
 
-    # Use Tuple `node_indices` and `evaluate_index` to copy values
-    # from the correct face and in the correct orientation
+    i_node_start, i_node_step = index_to_start_step(
```

Comment:
Since there is only one index, I think we can drop the `_node`, but we can also leave it for symmetry to the primary/secondary case.

Reply:
I prefer to keep it since we would get into trouble with `i` in the loop below.

Reply:
Ah, I missed that. Of course, it all makes sense now as it is.